wire-ios-data-model/Tests/Source/Model/Observer/ConversationObserverTests.swift

```diff
@@ -553,6 +554,22 @@ final class ConversationObserverTests: NotificationDispatcherTestBase {
 
     }
 
+    func testThatItNotifiesTheObserverOfChangeOneOnOneUser() {
+        // given
+        let otherUser = ZMUser.insertNewObject(in: self.uiMOC)
+        let conversation = ZMConversation.insertNewObject(in: self.uiMOC)
+        conversation.conversationType = ZMConversationType.oneOnOne
+        uiMOC.saveOrRollback()
```

Comment:
Should we stick with the Apple API of `try uiMoc.save()` to keep it simple and less prone to errors?

Reply:
I suppose it makes to in tests at least.